diff --git a/node_modules/react-native/scripts/cocoapods/jsengine.rb b/node_modules/react-native/scripts/cocoapods/jsengine.rb
index 6551ef3e50..fac2fd911a 100644
--- a/node_modules/react-native/scripts/cocoapods/jsengine.rb
+++ b/node_modules/react-native/scripts/cocoapods/jsengine.rb
@@ -32,7 +32,7 @@ def setup_hermes!(react_native_path: "../node_modules/react-native", fabric_enab
     pod 'React-jsi', :path => "#{react_native_path}/ReactCommon/jsi"
     pod 'hermes-engine', :podspec => "#{react_native_path}/sdks/hermes-engine/hermes-engine.podspec"
     pod 'React-hermes', :path => "#{react_native_path}/ReactCommon/hermes"
-    pod 'libevent', '~> 2.1.12'
+    pod 'libevent', :podspec => './custom-podspecs/libevent.podspec'
 end

 def add_copy_hermes_framework_script_phase(installer, react_native_path)
