--- a/node_modules/@exodus/react-native-threads/android/build.gradle
+++ b/node_modules/@exodus/react-native-threads/android/build.gradle
@@ -1,11 +1,12 @@

 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
+        google()
     }

     dependencies {
-        classpath 'com.android.tools.build:gradle:2.2.3'
+        classpath 'com.android.tools.build:gradle:7.3.1'
     }
 }
